test/models/activity_test.rb

```diff
@@ -0,0 +1,5 @@
+require 'test_helper'
+
+class ActivityTest < ActiveSupport::TestCase
+
+end
```

Comment:
No Doubt
I'm just a :girl: in the :earth_americas: That's all that you'll let me be.